build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:2.3.3'
         classpath 'com.dicedmelon.gradle:jacoco-android:0.1.1'
-
+        classpath 'com.github.ben-manes:gradle-versions-plugin:0.15.0'
```

Comment:
Explanation?

Reply:
Please see the commit message for that commit.